src/index.js

```diff
@@ -0,0 +1,174 @@
+//--API--//
+
+function getForecast(coordinates) {
+    console.log(coordinates);
+    let apiKey = "f2a7bf87b777299f2d3968c89592e347";
+    let apiUrl = `https://api.openweathermap.org/data/2.5/onecall?lat=${coordinates.lat}&lon=${coordinates.lon}&appid=${apiKey}&units=metric`;
+    console.log(apiUrl);
+
+    axios.get(apiUrl).then(displayForecast);
+}
+
+function getDataTemp(response) {
+
+    document.querySelector("#city").innerHTML = response.data.name;
+    celciusTemperature = response.data.main.temp;
+    document.querySelector("#res").innerHTML = Math.round(celciusTemperature);
+
+    document.querySelector("#hum").innerHTML = response.data.main.humidity;
+    document.querySelector("#wind").innerHTML = Math.round(response.data.wind.speed);
+    console.log(response);
+    document.querySelector("#description").innerHTML = response.data.weather[0].description;
+
+    document.querySelector("#icon").setAttribute("src", `http://openweathermap.org/img/wn/${response.data.weather[0].icon}@2x.png`);
+    document.querySelector("#icon").setAttribute("alt", response.data.weather[0].description);
+
+    getForecast(response.data.coord);
+}
+
+getData("Kyiv");
+
+function getData(city) {
+    let apiKey = "f2a7bf87b777299f2d3968c89592e347";
+    let apiURL = `https://api.openweathermap.org/data/2.5/weather?q=${city}&appid=${apiKey}&units=metric`;
```

Comment:
the same for apiURL, can be a variable or even a method if you need to provide some data, like
``` javascript
apiUrl = (city, apiKey) => `https://url/${city}/${apiKey}](https://api.openweathermap.org/data/2.5/weather?q=${city}&appid=${apiKey}&units=metric`
```